### ISSUE TITLE
Surface interferes with TouchableOpacity

### ISSUE THREAD

**Issue by BogdanArdelean** [tokkozhin/react-native-neomorph-shadows#24]
Hello!

While developing our app, we found an issue regarding the ART Surface on which the shadows are drawn for Neomorph. 

The functionality we tried to achieve was some Neomorph buttons. We implemented them with the Neomorph component and TouchableOpacity as a child. 

The problem was that when we laid out multiple Neomorph buttons next to each other, some of them didn't catch the onPress event, basically becoming untouchable.
 
After investigating with the Inspector tool, we found out that the ART Surface on which the OuterShadowART is drawn may be significantly bigger than the Neomorph component, thus rendering over neighbouring components and preventing the TouchableOpacity to be pressed.

As a fix, we changed the renderOuter function in Neomorph.js from

```javascript
const renderOuter = () => {
      if (useArt || Platform.OS !== 'ios') {
        return (
          <>
            <OuterShadowART {...styleDark} />
            <OuterShadowART {...styleLight} />
          </>
        );
      } 
...
````

to 

```javascript
const renderOuter = () => {
       if (useArt || Platform.OS !== 'ios') {
           return (
             <View pointerEvents={'none'}>
               <OuterShadowART {...styleDark} />
               <OuterShadowART {...styleLight} />
             </View>
           );
       }
...
```
This prevents the View in which the shadows are drawn to capture pointer events and everything works as expected.

We only tested on Android, but this may also be a problem on iOS. As a rule o thumb, I think that all the Views/<> on which the shadows are drawn should have `pointerEvents={'none'}`.

What do you think?

**Comment by BogdanArdelean** [comment]
The problem exists on iOS only when using ART. 
RN version: ```0.61.5```

**Comment by tokkozhin** [comment]
Hi! Yes, you're right! It's no conflict with other code and I'll add it in next version. 
Thx for contributing!

**Comment by tokkozhin** [comment]
Implemented in a new v1.1.0! 🎉
